apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -10,6 +10,34 @@
 
 namespace A8C\FSE\Coming_soon;
 
+$blog_locale = function_exists( 'get_blog_lang_code' ) ? get_blog_lang_code() : get_locale();
+
+function get_redirect_to() {
+
+	// redirect_to is probably never set so this is almost pointless
+	if ( ! empty( $_REQUEST['redirect_to'] ) ) {
+		return rawurlencode( $_REQUEST['redirect_to'] );
+	}
+
+	if ( ! isset( $_SERVER['HTTP_HOST'] ) || ! isset( $_SERVER['REQUEST_URI'] ) ) {
```

Comment:
hmm when would this case happen?

Reply:
This was in response to a PHPCS error of `possibly undefined superglobal` that wouldn't let me commit until I'd addressed it. 

I agree that it's probably not needed. I'll see if I can squeeze around it